pkg/util/util.go

```diff
@@ -39,6 +33,11 @@ import (
 	"strconv"
 	"strings"
 	"time"
+
+	"github.com/juicedata/juicefs-csi-driver/pkg/config"
```

Comment:
Can you move these formatting and adding Context into another PR, without any behavior changes, so we can have a small PR and focus on the actual changes?

Reply:
Yes, formatting can be moved to another PR, but adding Context is the main change actually. (I supplement more explanation in the description)

Reply:
We can pass context.TODO around without actually using it, then we can get rid most of changes out of this  PR, and focus on the actual changed behaviors. Otherwise we may miss some important changes.

Reply:
Ok, I'm going to create a new PR to pass context around.